spec.html

```diff
@@ -5624,19 +5646,6 @@ <h1>IsRegExp ( _argument_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-isstringprefix" aoid="IsStringPrefix">
-      <h1>IsStringPrefix ( _p_, _q_ )</h1>
```

Comment:
For readability, we could leave this with an implementation that just passes `0` as the third parameter to `StringIncludesAt`. Not sure if it's worth it.

Reply:
It's maybe just for me, but parameter order of [`IsStringPrefix`](https://tc39.es/ecma262/#sec-isstringprefix) isn't immediately clear: is first parameter a string or a prefix?

Reply:
I feel the same way about `StringIncludesAt`.

Reply:
It would be great if we came up with a better name. `StringMatch`?

Reply:
I don't really think the name will ever be sufficient. I will always need to take a quick look at the parameter names or the summary. Or infer it from context at the usage site.